scalardl-test/verification-config.toml

```diff
@@ -24,6 +24,8 @@
   is_verification = true
   num_accounts = 10
   population_concurrency = 32
+  checker_retry_interval_millis = 1000
```

Comment:
I don't like them because it sounds like retrying the checker process, but it actually only retires the read balances, 
in the checker so what the variable sounds like and what it does are not consistent.
So, to make them consistent, retry the whole checker or make the variables specific for retrying read balances. I was suggesting retrying the whole checker because it makes more sense to me.

Anyway, I think it should be refactored at some point, but probably It would take too much more time for you so I would approve it for now.